meson.build

```diff
@@ -2672,9 +2685,9 @@ exes += exe
 
 # XXX: HAVE_OPENAT && HAVE_DIRFD
 exe = executable(
-  'test_procutils',
-  'lib/procutils.c',
-  c_args : ['-DTEST_PROGRAM_PROCUTILS'],
+  'test_procfs',
+  'lib/procfs.c',
+  c_args : ['-DTEST_PROGRAM_PROCFS'],
   include_directories : dir_include)
```

Comment:
This now also needs to link to `lib_common`

Reply:
Thank you!
Now all tests are passed.